views/tx.tmpl

```diff
@@ -375,6 +377,9 @@
                     {{end}}
                 </tbody>
             </table>
+            {{else}}
+            <br>No votes in this block.</p>
```

Comment:
No votes in this *transaction*?

Reply:
But in general, I found the wording a bit confusing since a vote must always validate the previous block.  Really what this change intends to say is that this vote/ssgen/transaction contains no choices on known agendas